django/contrib/admin/static/admin/css/base.css

```diff
@@ -20,10 +20,15 @@ a:link, a:visited {
     text-decoration: none;
 }
 
-a:hover {
+a:focus, a:hover {
     color: #036;
 }
 
+a:focus {
+    outline: none;
```

Comment:
The admin has a discreet design. It aims at *not* carrying a distinct brand.

For this reason I agree that it's better to stick to the platform's conventions.